primehub/extras/templates/examples/admin/instancetypes.md

```diff
@@ -6,10 +6,10 @@
 | displayName | optional | string | display name for the instance type |
 | description | optional | string | |
 | global | optional | boolean | when an instance type is global, it could be seen for each group |
-| groups | optional | list of connected groups (dict) | please see the `connect` examples |
-| cpuLimit | required | float | the maximum vCPU quantity. For example: `1` or `1.0` means 1 vCPU and `0.5` means half of vCPU |
```

Comment:
Auto-filling feature should not change a field required or not. Please refer to other review comments